samarth641.md

```diff
@@ -0,0 +1,50 @@
+---
+name: "@samarth641"
+project: "Quadcopter Drone"
+---
+
+# Quadcopter Drone
+
+## Summary
+
+A drone is a small, unmanned aircraft that is typically controlled remotely by a pilot or through a pre-programmed flight path. 
+Drones are used for a wide range of purposes, including surveillance, photography, mapping, delivery, and scientific research.
+Building a drone can be an exciting and challenging project, as it involves a combination of engineering, design, and programming skills. 
+The goal is to create a device that is capable of stable, controlled flight, and that can perform a variety of tasks depending on its design and purpose.
+will be making this project with my friend Om tripathi .
+
+## Plan
+
+Step 1 We begin by gathering all the necessary parts and tools for assembling the drone. This includes the main body of the drone, the propellers, motors, battery, and any additional components that are specific to our particular drone model. We also gather a few tools, such as a screwdriver and a pair of pliers.
+
+Step 3 With all the necessary parts and tools on hand, we start the assembly process. First, we attach the motors to the body of the drone. Each motor is connected to a specific arm of the drone, and we use screws or bolts to secure them in place. We make sure that the motors are securely fastened, as they will be responsible for lifting the drone off the ground.
+
+Step 4 Next, we attach the propellers to the motors. The propellers have a specific orientation, and we make sure that they are facing the correct direction. We can usually tell which way the propellers should face by looking for a small arrow or line on the base of the propeller.
+
+Step 5 With the propellers in place, we install the battery and any additional electronics, such as a flight controller or GPS module. We make sure that all of these components are securely fastened and properly connected to the main body of the drone.
```

Comment:
will your propellers be 3d printed? which parts will you be printing?

also, do you plan on adding a GPS module? i don't think i see that in your budget?

Reply:
We will be printing the Body of the drone not the propellers. I have not included the gps module in the budget because I am already having one with me. So will be using that

Reply:
ahhh cool! and what do you mean by the body of the drone? the frame? i believe that this item within your budget looks like it provides the frame? https://www.flyrobo.in/dji-f450-quadcopter-frame-kit-with-integrated-pcb

also, i might have missed it but are you purchasing propellers? i don't think i see it in your budget?

and could you please add that you already have the gps module (and put $0) in your budget?

Reply:
I believe I forgot to include propellers in my budget. Sorry about that, I've added it now, along with the GPS module for $0, and updated the total. See what I have added to the budget is the frame of the drone where all the electric parts will be attached and the 3d model will be for covering the drone which will be the body of drone reposble for protecting the elctronic and making the drone more thermodynamic.

Reply:
no worries, thanks for pushing the changes!